packages/mtr.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Mtr < Package
+  description "mtr combines the functionality of the 'traceroute' and 'ping' programs in a single network diagnostic tool."
+  homepage 'https://www.bitwizard.nl/mtr/'
+  version '0.93'
+  source_url 'ftp://ftp.bitwizard.nl/mtr/mtr-0.93.tar.gz'
+  source_sha256 '229c673d637bd7dbb96471623785a47e85da0b1944978200c949994c1e6af10d'
+
+  binary_url ({
+    aarch64: '',
+     armv7l: '',
+     x86_64: '',
+       i686: ''
+  })
+  binary_sha256 ({
+    aarch64: '',
+     armv7l: '',
+     x86_64: '',
+       i686: '',
+  })
+
+  depends_on 'ncurses'
```

Comment:
Remove this. `ncurses` is a core package and is not required to be listed as a dependency.

Reply:
was removed in 2ef13a0703cace4b607b73fd2768e44d550f5131